### PR TITLE
Deploying AngularFire to Firebase docs fixes

### PR DESCRIPTION
**Pull request by exequiel09**
Fix typo and add some missing words

### CODE REVIEW

File: docs/7-deploying-angularfire-to-firebase.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ ng build -prod
 
 ### 1. Run Firebase init
 
-You must initliazie firebase hosting in order to deploy your application. In order to this run the `firebase init` command.
+You must initialize the firebase hosting in order to deploy your application. In order to do this, run the `firebase init` command.
```
Comment (davideast) [inline docs/7-deploying-angularfire-to-firebase.md]: I'd prefer "initialize Firebase Hosting".

Comment (davideast) [inline docs/7-deploying-angularfire-to-firebase.md]: No comma required here.

**Comment by exequiel09** [comment]
@davideast All requested changes have been applied. Kindly check if it is okay. Thanks.

**Comment by davideast** [comment]
Perfect! Thanks @exequiel09!

**Pull request merged**